### PR TITLE
Release 0.8.2

### PR DESCRIPTION
**Pull request by javisantana**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -119,5 +120,5 @@ test_extension_new:
 
 legacy_tests: legacy_regress 
 
-installcheck: legacy_tests test_extension_new test_organization
+installcheck: test_organization
```
Comment (rochoa) [inline Makefile]: Why did you remove those other suites?

Comment (javisantana) [inline Makefile]: by mistake, I added them later

Comment (rochoa) [inline Makefile]: Oh, I see. Sorry! :snail:
